packages/core/src/electron-main/electron-main-application.ts

```diff
@@ -243,7 +247,15 @@ export class ElectronMainApplication {
         if (browserWindow) {
             this.saveWindowState(browserWindow);
         } else {
-            console.warn(`no BrowserWindow with id: ${webContents.id}`);
+            console.warn(`no BrowserWindow wit  "theia": {
```

Comment:
Typo: "wit".

Reply:
Oops, that actually looks like a vi-accident of mine. Reverted to original line in a follow-up commit.